Makefile

```diff
@@ -395,7 +395,7 @@ compose-%:
 	echo "    started compose-$*" >> $(LOGFILE)
 	cat $(DIR)/lib/compose/docker-compose.yml $(DIR)/lib/logging/$(DOCKER_LOG_DRIVER)/docker-compose.yml \
 		$(DIR)/lib/$*/docker-compose.yml \
-		| docker-compose -f - up -d $(SERVICE)
```

Comment:
The service variable is needed in the case that your docker-compose has multiple containers/services.

Reply:
Yeah, but it causes problems if what you want is to spin up all of the containers in docker-compose (as is the case for opa)